src/api/TinRoll.Api/Startup.cs

```diff
@@ -29,14 +29,15 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
             services.AddDbContext<TinRollContext>(
-                options => options.UseSqlServer("Server=(localdb)\\mssqllocaldb;Database=TinRollDb;Trusted_Connection=True;MultipleActiveResultSets=true"));
+                options => options.UseSqlServer(@"Server=DESKTOP-P0PO0N5\MSSQLLOCALDB;Database=TinRollDb;Trusted_Connection=True;MultipleActiveResultSets=true"));
```

Comment:
Just fyi, this will be moving to a configuration file soon. I'll let you know the details whenever that changes.